test/tests/interfaces/pbs_partition.py

```diff
@@ -0,0 +1,297 @@
+                                id="Q1", runas=TEST_USER)
+        except PbsManagerError as e:
+            # self.assertEquals(e.rc, 15007)
+            # The above code has to be uncommented when the PTL framework
+            # bug PP-881 gets fixed
+            self.assertTrue(msg1 in e.msg[0], msg2)
+
+    def test_set_partition_to_routing_queue(self):
+        """
+        Test to check the set of partition attribute on routing queue
+        """
+        msg1 = "Can not assign a partition to route queue"
+        msg2 = "checking the qmgr error message"
+        try:
+            self.partition_attr(mgr_cmd="MGR_CMD_CREATE",
+                                q_type='route', enable='False', start='False')
+        except PbsManagerError as e:
+            # self.assertEquals(e.rc, 15217)
+            # The above code has to be uncommented when the PTL framework
+            # bug PP-881 gets fixed
+            self.assertTrue(msg1 in e.msg[0], msg2)
+        self.server.manager(
+            MGR_CMD_CREATE, QUEUE, {
+                'queue_type': 'r'}, id='Q1')
+        try:
+            self.server.manager(MGR_CMD_SET,
+                                QUEUE, {'partition': "P1"},
+                                id="Q1")
+        except PbsManagerError as e:
+            # self.assertEquals(e.rc, 15007)
+            # The above code has to be uncommented when the PTL framework
+            # bug PP-881 gets fixed
+            self.assertTrue(msg1 in e.msg[0], msg2)
+
+    def test_modify_queue_with_partition_to_routing(self):
+        """
+        Test to check the modify of execution queue to routing when
+        partition attribute is set
+        """
+        self.partition_attr(mgr_cmd="MGR_CMD_CREATE")
+        msg1 = ("Route queues are incompatible "
+                "with the partition attribute queue_type")
+        msg2 = "checking the qmgr error message"
+        try:
+            self.server.manager(MGR_CMD_SET,
+                                QUEUE, {'queue_type': "r"},
+                                id="Q1")
+        except PbsManagerError as e:
+            # self.assertEquals(e.rc, 15218)
+            # The above code has to be uncommented when the PTL framework
+            # bug PP-881 gets fixed
+            self.assertTrue(msg1 in e.msg[0], msg2)
+
+    def test_set_partition_without_queue_type(self):
+        """
+        Test to check the set of partition attribute on queue
+        with not queue_type set
+        """
+        self.server.manager(MGR_CMD_CREATE,
+                            QUEUE, {'partition': "P1"},
+                            id="Q1", expect=True)
+        self.server.manager(MGR_CMD_SET,
+                            QUEUE, {'partition': "P2"},
+                            id="Q1", expect=True)
+        self.server.manager(MGR_CMD_SET,
+                            QUEUE, {'queue_type': "execution"},
+                            id="Q1", expect=True)
```

Comment:
Why not use partition_attr() to create the above 3 queues?

Reply:
Done